packages/api/src/routes/internal/index.ts

```
@@ -62,6 +62,8 @@ import organizationRoutes from "./medical/organization";
 import patientRoutes from "./medical/patient";
 import tcmEncounter from "./medical/tcm-encounter";
 import suspectRoutes from "./medical/suspect";
+import questRoutes from "./integration/quest";
```

Biome (2.1.2)

[error] Shouldn't redeclare 'questRoutes'. Consider to delete it or rename it.

'questRoutes' is defined here:


(lint/suspicious/noRedeclare)